src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -500,7 +497,14 @@ static void Main()
         }
     }
 }";
-            CreateCompilation(source).VerifyDiagnostics();
+            CreateCompilation(source).VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](start = 38, length = 17)

:memo: this scared me (looked like a breaking change), but this is actually restoring the existing behavior (`master` produces `error CS1674: 'C2': type used in a using statement must be implicitly convertible to 'System.IDisposable'`)